src/abstractdataframe/selection.jl

```diff
@@ -17,6 +17,31 @@
 
 A type used for selection operations to signal that the wrapped function should
 be applied to each element (row) of the selection.
+
+Note that `ByRow` always wraps retun values of `fun` into a vector.
```

Comment:
You mean that they are not destructured into columns? By the way, maybe we should reserve this in case we want to allow creating multiple columns by returning a named tuple in the future?

Also:
```suggestion
Note that `ByRow` always wraps return values of `fun` into a vector.
```

Reply:
Yes - they are not destructed. I have expanded on why I did it below. But please comment there if you feel otherwise.